{"uid":"65e01042669b8dc9","name":"Checks that host card button text includes counter which changed after click & check value reverted after reload","fullName":"Checks that host card button text includes counter which changed after click & check value reverted after reload","historyId":"cc4fe52be36f18b1502261912a0b28ca","time":{"start":1681666568646,"stop":1681666569785,"duration":1139},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"\"before each\" hook","time":{"start":1681666568646,"stop":1681666568824,"duration":178},"status":"passed","steps":[{"name":"visit http://localhost:4173","time":{"start":1681666568736,"stop":1681666568824,"duration":88},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"http://localhost:4173 request","time":{"start":1681666568824,"stop":1681666568824,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":2,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"get button","time":{"start":1681666568825,"stop":1681666569784,"duration":959},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1681666569784,"stop":1681666569784,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[{"uid":"88e55fed673d650f","name":"e2e/tests/runAll.cy.ts:2023-04-16T17:36:09.480Z:attempt-1","source":"88e55fed673d650f.png","type":"image/png","size":152084},{"uid":"3da7591c4f6a0178","name":"video recording","source":"3da7591c4f6a0178.mp4","type":"video/mp4","size":3489871}],"parameters":[],"stepsCount":19,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.tests.runAll.cy.ts"},{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks host app"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c6feddc19fb43f7b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681666567794,"stop":1681666568643,"duration":849}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":33,"broken":0,"skipped":0,"passed":111,"unknown":0,"total":144},"items":[{"uid":"c89bdc28b0a034ed","reportUrl":"https://module-federation.github.io/module-federation-examples/710//#testresult/c89bdc28b0a034ed","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681596538160,"stop":1681596539643,"duration":1483}},{"uid":"53e5e984dc1cb2df","reportUrl":"https://module-federation.github.io/module-federation-examples/707//#testresult/53e5e984dc1cb2df","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681537407893,"stop":1681537409260,"duration":1367}},{"uid":"7abaeecf6b200906","reportUrl":"https://module-federation.github.io/module-federation-examples/704//#testresult/7abaeecf6b200906","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681519379108,"stop":1681519380406,"duration":1298}},{"uid":"c53f1a0de26e795f","reportUrl":"https://module-federation.github.io/module-federation-examples/696//#testresult/c53f1a0de26e795f","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681338406162,"stop":1681338407675,"duration":1513}},{"uid":"9da384a51be27b4b","reportUrl":"https://module-federation.github.io/module-federation-examples/695//#testresult/9da384a51be27b4b","status":"passed","time":{"start":1681324159715,"stop":1681324161346,"duration":1631}},{"uid":"1ec76d82d7290fee","reportUrl":"https://module-federation.github.io/module-federation-examples/694//#testresult/1ec76d82d7290fee","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681323463620,"stop":1681323465358,"duration":1738}},{"uid":"4cf3284077aa58f","reportUrl":"https://module-federation.github.io/module-federation-examples/693//#testresult/4cf3284077aa58f","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681299813298,"stop":1681299814759,"duration":1461}},{"uid":"b8a1f575ddf275c6","reportUrl":"https://module-federation.github.io/module-federation-examples/687//#testresult/b8a1f575ddf275c6","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681214579859,"stop":1681214581479,"duration":1620}},{"uid":"dd9107bba3a0b8c7","reportUrl":"https://module-federation.github.io/module-federation-examples/686//#testresult/dd9107bba3a0b8c7","status":"passed","time":{"start":1681146969224,"stop":1681146971308,"duration":2084}},{"uid":"31aec9315446aafd","reportUrl":"https://module-federation.github.io/module-federation-examples/685//#testresult/31aec9315446aafd","status":"passed","time":{"start":1681146969224,"stop":1681146971308,"duration":2084}},{"uid":"b7748d53f1892938","reportUrl":"https://module-federation.github.io/module-federation-examples/684//#testresult/b7748d53f1892938","status":"passed","time":{"start":1681136331783,"stop":1681136333374,"duration":1591}},{"uid":"166ec0653dc9ff1a","reportUrl":"https://module-federation.github.io/module-federation-examples/683//#testresult/166ec0653dc9ff1a","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681099509793,"stop":1681099511001,"duration":1208}},{"uid":"bd356ea915c5a953","reportUrl":"https://module-federation.github.io/module-federation-examples/682//#testresult/bd356ea915c5a953","status":"passed","time":{"start":1681098139743,"stop":1681098141103,"duration":1360}},{"uid":"b34e314fef67ec57","reportUrl":"https://module-federation.github.io/module-federation-examples/671//#testresult/b34e314fef67ec57","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680987656232,"stop":1680987657934,"duration":1702}},{"uid":"b3b898a81a89d07","reportUrl":"https://module-federation.github.io/module-federation-examples/665//#testresult/b3b898a81a89d07","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680909807277,"stop":1680909808324,"duration":1047}},{"uid":"f9c8b2b63bcb90d2","reportUrl":"https://module-federation.github.io/module-federation-examples/661//#testresult/f9c8b2b63bcb90d2","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680834529867,"stop":1680834531699,"duration":1832}},{"uid":"f288bd577ba86cc6","reportUrl":"https://module-federation.github.io/module-federation-examples/657//#testresult/f288bd577ba86cc6","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680806544791,"stop":1680806546446,"duration":1655}},{"uid":"14d0d8d7b576444c","reportUrl":"https://module-federation.github.io/module-federation-examples/630//#testresult/14d0d8d7b576444c","status":"passed","time":{"start":1680739145799,"stop":1680739147825,"duration":2026}},{"uid":"fbfaf87e59dda192","reportUrl":"https://module-federation.github.io/module-federation-examples/619//#testresult/fbfaf87e59dda192","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680701548016,"stop":1680701550065,"duration":2049}},{"uid":"a693d1bbdbd8dd96","reportUrl":"https://module-federation.github.io/module-federation-examples/616//#testresult/a693d1bbdbd8dd96","status":"passed","time":{"start":1680695672079,"stop":1680695673494,"duration":1415}}]},"tags":[]},"source":"65e01042669b8dc9.json","parameterValues":[]}